src/helper/events.ts

```diff
@@ -7,9 +7,9 @@ import { MynahEventNames } from '../static';
 import { generateUID } from './guid';
 
 export const cancelEvent = (event: Event): boolean => {
-  event.preventDefault();
-  event.stopPropagation();
-  event.stopImmediatePropagation();
+  event.preventDefault?.();
```

Comment:
Doesn't all events have these functions?

Reply:
Unfortunately not. It can be overridden during generation